assets/scripts/airport.js

```diff
@@ -95,6 +98,7 @@ zlsa.atc.ArrivalDefault = Fiber.extend(function(base) {
         airline:   choose_weight(this.airlines),
         altitude:  altitude,
         heading:   this.heading,
+        fixes:     this.fixes.slice(),
```

Comment:
A good point - not knowing the whole code base, I guess I was being slightly paranoid not knowing what other code might do with the options. The defensive approach was to clone it and prevent any means of it being modified.